functionary/core/models/function.py

```diff
@@ -14,8 +14,6 @@ class Function(models.Model):
         display_name: optional display name
         description: more details about the function
         schema: the function's OpenAPI definition
-        active: whether this function is the one that is taskable. False indicates that
```

Comment:
I decided to remove the active field for now.  Not related to the main thrust of this PR, but I think tracking the function history is a needless complication for right now and possibly we can do what I was envisioning without it.